### ISSUE TITLE
Pairing Sessions

### ISSUE THREAD

**Issue by lenary** [rubypair/rubypair#14]
So I know @elight has a google calendar where you can book office hours with him. 

I think it would be cool if we could have a system where I say which hours i am available, and then someone can come in and say "i want that hour" or "i want that day" and everything is matched up including the spreading of details to both parties

Maybe i'm just mad though. I have other ideas that are as mad.

**Comment by razielgn** [comment]
Yeah this is great. Please, keep the mad ideas coming: I'm very curious. :)

**Comment by elight** [comment]
I hugely want this feature. However, scheduling is an application unto itself, e.g. see http://tundle.me..

There already exist a few apps that do this.  However, I'm not sure that any of them are adequate to the task.  Hell, it could be a viable niche market to enter into with a startup.

Going to see if I can get Avdi to remark on this issue. We've talked about apps that fill this need. He's done more research here than I.

**Comment by avdi** [comment]
They all suck.

YouCanBookMe is the best I've seen so far, and it kinda sucks.

Um... I'll have to think about this. I honestly don't have a lot of constructive thoughts about it at the moment.

**Comment by elight** [comment]
Damn. Didn't get to finish that email. Meant to say "considering it as a potential business but still considering how to approach it." It would certainly scratch an itch for me. And I have more uses for such a beast than just pairing.

**Comment by kesor** [comment]
This is related to the "Right now" feature.

**Comment by kesor** [comment]
How about you just allow people to stick a http://tungle.me account button on their names next to their github/twitter buttons?
(you have a typo in the url btw)

Maybe even shoot tungle.me guys an email and ask if they have any special APIs that can be used.

**Comment by elight** [comment]
You're assuming that we didn't already look at tungle.me.  They once
offered an API but no longer.

On Oct 22, 2011, at 11:57 PM, Evgeny Zislis
reply@reply.github.com
wrote:

**Comment by kesor** [comment]
I noticed they allow to jump into a calendar request thing, kinda like
your open google calendar. But it works with more calendar services,
allows login using twitter/facebook/google. So a button next to each
developer that sends the user to his tungle.me page would be just the
thing.

Or in your case, a button to send to google calendar.

Maybe just an open URL that a user can fill in his profile with
suggestions or explanations how to set it up in either google or
tungle.

On 23 באוק 2011, at 06:00, Evan David Light
reply@reply.github.com
wrote:

**Comment by elight** [comment]
Good idea.

On Oct 23, 2011, at 12:05 AM, Evgeny Zislis
reply@reply.github.com
wrote:

**Comment by gaustin** [comment]
So, tungle.me is dead. Anyone tried [Schedulicity](http://www.schedulicity.com)?
